byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -52,4 +60,39 @@ private String formatLocation(PlaceType location) {
     place = place.replace('_', ' ');
     return place;
   }
+
+  /**
+   * Temporary setup to find entity with hard-coded id value. Set their interests.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query person =
+      new Query("user")
+        .setFilter(new FilterPredicate("id", FilterOperator.EQUAL, 2452));
+    PreparedQuery pq = datastore.prepare(person);
+    Entity userEntity = pq.asSingleEntity();
+    if(userEntity != null) {
+      String interestsAsString = request.getParameter("data");
+      addInterestsForUser(interestsAsString, userEntity, datastore);
+    } else {
+      response.setStatus(response.SC_NOT_FOUND);
```

Comment:
I don't think you should hardcode the status, because you can't guarantee that it will be a 404 error.

Reply:
I set the status because it would indicate a user wasn't found in datastore. What do you recommend I do to handle this case? I set it as a not found error, albeit not the same thing.

Reply:
I added a comment to show the entity must exist and cannot be null since it's made beforehand, and referenced with a specific unique trip id to guarantee it's found and a single entity. Fixed